### PR TITLE
Kinesis streams

### PR DESCRIPTION
**Pull request by Lowercases**
Adds a kinesis streams module, modelled after s3.py.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,12 @@
+DEPRECATED
```
[the remark this answers is not in the document]
Comment (russellballestrini) [reply README.rst]: +1

File: stacker_blueprints/kinesis.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    def create_template(self):
+        t = self.template
+        variables = self.get_variables()
+
+        streams = variables["Streams"]
+        stream_ids = []
+
+        for stream in streams:
+            s = t.add_resource(stream)
+            t.add_output(Output("%sStreamId" % stream.title, Value=Ref(s)))
```
[the remark this answers is not in the document]
Comment (russellballestrini) [reply stacker_blueprints/kinesis.py]: I prefer the lowercase versions `.ref()`.

File: stacker_blueprints/kinesis.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+from awacs.aws import Allow, Policy, Statement
+import awacs.kinesis
+
+
+def stream_arn(stream_id):
+    return Sub("arn:aws:kinesis:${AWS::Region}:${AWS::AccountId}:"
+               "stream/${Stream}", Stream=stream_id)
+
+
+def read_only_kinesis_stream_policy_statements(stream_arns):
```
Comment (phobologic) [inline stacker_blueprints/kinesis.py]: Any reason to not put this in `policies.py`?

File: stacker_blueprints/kinesis.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (phobologic) [reply stacker_blueprints/kinesis.py]: https://github.com/cloudtools/troposphere/blob/master/troposphere/__init__.py#L312

**Pull request merged**